src/program/snabbvmx/doc/README.troubleshooting.md

```diff
@@ -370,12 +400,12 @@ Usage: check [-r] CONF V4-IN.PCAP V6-IN.PCAP V4-OUT.PCAP V6-OUT.PCAP
 
 Parameters:
 
-- **CONF**: SnabbVMX (icmp_snabbvmx-lwaftr-xe.cfg) or lwaftr (icmp_snabbvmx-lwaftr-xe1.conf) configuration file.
-- **V4-IN.PCAP**: Incoming IPv4 packets (from Internet).
-- **V6-IN.PCAP**: Incoming IPv6 packets (from b4).
-- **V4-OUT.PCAP**: Outgoing IPv4 packets (to Internet, decapsulated).
-- **V6-OUT.PCAP**: Outgoing IPv6 packets (to b4, encapsulated)
-- **[COUNTERS.LUA]**: Lua file with counter values. Will be regenerated via [-r] parm
+- CONF: SnabbVMX (icmp_snabbvmx-lwaftr-xe.cfg) or lwaftr (icmp_snabbvmx-lwaftr-xe1.conf) configuration file.
+- V4-IN.PCAP : Incoming IPv4 packets (from Internet).
+- V6-IN.PCAP : Incoming IPv6 packets (from b4).
+- V4-OUT.PCAP : Outgoing IPv4 packets (to Internet, decapsulated).
+- V6-OUT.PCAP : Outgoing IPv6 packets (to b4, encapsulated)
+- [COUNTERS.LUA] : Lua file with counter values. Will be regenerated via [-r] parm
```

Comment:
Other bullets list in the manual with the same pattern (description term: description) have the description term marked as bold. For instance, in the section "Components Overview". It's not that I don't like the change, but I prefer to keep the same style in all the document.